params/config.go

```diff
@@ -166,7 +166,7 @@ type UpgradeConfig struct {
 	// Config for optional timestamps that enable network upgrades.
 	// Note: if OptionalUpgrades is specified in the JSON all previously activated
 	// forks must be present or upgradeBytes will be rejected.
-	OptionalNetworkUpgrades *OptionalNetworkUpgrades `json:"networkUpgrades,omitempty"`
+	OptionalNetworkUpgrades *OptionalNetworkUpgrades
```

Comment:
This is needed, as far as I did my research, so the JSON decoder (from the local config) will flatten and serialize their children instead (`OptionalNetworkUpgrades` has instructions on how to deserialize the JSON file). 

TODO: Write tests on how to parse the config